apps/charterafrica/src/pages/index.page.js

```diff
@@ -42,6 +42,35 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
     fallbackLocale: defaultLocale,
   });
 
+  const { docs: homePage } = await payload.findPage("home", "pages", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  const { blocks } = homePage[0] ?? {
+    blocks: [],
+  };
+
+  const spotlightItems = blocks
+    .filter((block) => block.blockType === "spotlight")[0]
+    ?.items.map((spotlight) => {
+      const { item, ...rest } = spotlight;
+      const formattedItem = {
+        ...item,
+        image: {
+          src: item.image.url,
+          alt: item.image.alt,
+        },
+        date: new Date(item.date).toLocaleDateString(),
```

Comment:
```suggestion
        date: new Date(item.date).toLocaleDateString(locale, { year: 'numeric', month: 'short', day: 'numeric' }),
```

Or what format are we supposed to display the dates in?